frontend/server/src/Controllers/Course.php

```diff
@@ -3716,24 +3716,11 @@ private static function getAssignmentDetailsForTypeScript(
 
         // Get scoreboard;
         $scoreboard = new \OmegaUp\Scoreboard(
-            new \OmegaUp\ScoreboardParams([
-                'alias' => $assignment->alias,
-                'title' => $assignment->name,
-                'problemset_id' => $assignment->problemset_id,
-                'start_time' => $assignment->start_time,
-                'finish_time' => $assignment->finish_time,
-                'acl_id' => $assignment->acl_id,
-                'group_id' => $course->group_id,
-                'admin' => (
-                    \OmegaUp\Authorization::isCourseAdmin(
-                        $r->identity,
-                        $course
-                    ) ||
-                    \OmegaUp\Authorization::canCreatePublicCourse(
-                        $r->identity
-                    )
-                ),
-            ])
+            \OmegaUp\ScoreboardParams::fromAssignment(
+                $assignment,
+                intval($course->group_id),
+                /*showAllRuns*/true
+            )
         );
```

Comment:
`show_all_runs` era falso antes: se puede quedar como falso? también, falta el parámetro `'admin'`: se puede modificar después de haber creado el objeto?

```suggestion
        $params = \OmegaUp\ScoreboardParams::fromAssignment(
            $assignment,
            intval($course->group_id),
            /*showAllRuns*/false
         );
         $params->admin = (
           \OmegaUp\Authorization::isCourseAdmin(
                $r->identity,
                $course
            ) ||
            \OmegaUp\Authorization::canCreatePublicCourse(
                $r->identity
            )
        );
        $scoreboard = new \OmegaUp\Scoreboard($params);
```